gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -107,48 +106,42 @@ private void addPartitionInfoToBrokerInfo(
             });
   }
 
-  @Override
   public void activateJobs(
       final ActivateJobsRequest request,
-      final StreamObserver<ActivateJobsResponse> responseObserver) {
-    activateJobsHandler.activateJobs(
-        request, ErrorMappingStreamObserver.ofStreamObserver(responseObserver));
+      final ServerStreamObserver<ActivateJobsResponse> responseObserver) {
+    activateJobsHandler.activateJobs(request, responseObserver);
   }
 
-  @Override
   public void cancelWorkflowInstance(
       final CancelWorkflowInstanceRequest request,
-      final StreamObserver<CancelWorkflowInstanceResponse> responseObserver) {
+      final ServerStreamObserver<CancelWorkflowInstanceResponse> responseObserver) {
     sendRequest(
         request,
         RequestMapper::toCancelWorkflowInstanceRequest,
         ResponseMapper::toCancelWorkflowInstanceResponse,
-        ErrorMappingStreamObserver.ofStreamObserver(responseObserver));
+        responseObserver);
   }
 
-  @Override
   public void completeJob(
       final CompleteJobRequest request,
       final StreamObserver<CompleteJobResponse> responseObserver) {
```

Comment:
Why are some methods are changed to use the new interface and some not?

Reply:
Only the activateJobsHandler really needs the new type actually...but it makes sense to change all methods to accept it.